src/Synapse/Synapse/Common/HelpMessages.cs

```diff
@@ -88,7 +88,7 @@ public static class HelpMessages
 
         public const string DisableAutoPause = "Indicates whether Auto-pause should be disabled.";
 
-        public const string AutoPauseDelayInMinute = "Number of minutes idle. This parameter must be specified when Auto-pause is enabled.";
+        public const string AutoPauseDelayInMinute = "Number of minutes idle. This parameter can be specified when Auto-pause is enabled. The default value will be [15] if it is not specified manually.";
```

Comment:
You can use a formatted string here.
```suggestion
        public const string AutoPauseDelayInMinute = $"Number of minutes idle. This parameter can be specified when Auto-pause is enabled. The default value will be {SynapseConstants.DefaultAutoPauseDelayInMinute} if it is not specified manually.";
```

Reply:
const interpolated string is unsupporting here, and the help messages must be const strings, so change it to concatenation.